src/Lykke.Service.Assets/Attributes/KeyFormatAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace Lykke.Service.Assets.Attributes
+{
+    public class KeyFormatAttribute : ValidationAttribute
+    {
+        private const string Pattern = "^[A-Za-z][A-Za-z0-9\\s]{2,62}$";
```

Comment:
Regex can be static object. In theory it save a little time on the pattern compilation for every validation

Reply:
Yes, right. But this validation happens rarely.